src/heretic/g_game.c

```diff
@@ -1510,11 +1513,65 @@ void G_SecretExitLevel(void)
     gameaction = ga_completed;
 }
 
+// [crispy] Write level statistics upon exit
+void G_WriteLevelStat(void)
+{
+    static FILE *fstream = NULL;
+
+    int i, playerKills = 0, playerItems = 0, playerSecrets = 0;
+    int exitHours, exitMinutes;
+    float exitTime, exitSeconds;
+
+    char levelTimeString[16];
+    
+    if (fstream == NULL)
```

Comment:
Extra parentheses, please.

Reply:
Parentheses? Like `if ((fstream == NULL))`? What does that do exactly?

Or do you mean brackets around the following line?

Reply:
I've added braces at least :^)

Reply:
Yeah, right. You know, in the German language it's all "Klammer". We have "runde Klammer" `(`, "eckige Klammer" `[`, "spitze Klammer" `<` and "geschweifte Klammer" `{`. However, I can't remember the individual words for them in  English. :wink: